app/src/tests/component-tests/Dashboard.test.js

```diff
@@ -23,4 +23,12 @@ describe('Dashboard functionality', () => {
         wrapper.find(IconButton).first().simulate('click');
         expect(wrapper.state().open).toBe(false);
     });
+
+    it('renders a Dashboard component with drawer and simulate selecting instruments', () => {
```

Comment:
Are you mocking the service call here? I don't see the change

Reply:
👍